diff --git a/setup.py b/setup.py
index 6c96175..f011870 100644
--- a/setup.py
+++ b/setup.py
@@ -3,39 +3,49 @@ __revision__ = "$Id$"
 
 import os, sys
 from setuptools import setup, find_packages
-from openalea.deploy.binary_deps import binary_deps
+#from openalea.deploy.binary_deps import binary_deps
 pj = os.path.join
 
 import versionreader
 
-from openalea.deploy.metainfo import read_metainfo
-metadata = read_metainfo('metainfo.ini', verbose=True)
-for key,value in metadata.iteritems():
-    exec("%s = '%s'" % (key, value))
+#from openalea.deploy import metainfo
+#read_metainfo = metainfo.read_metainfo
+#metadata = read_metainfo('metainfo.ini', verbose=True)
+#for key,value in metadata.iteritems():
+#    exec("%s = '%s'" % (key, value))
 
 
 # Setup script
-meta_version = version 
+#meta_version = version
 version = versionreader.getPGLVersionString()
-if meta_version != version:
-    print 'Warning:: Update the version in metainfo.ini !!'
-print pkg_name,': version =',version
+#if meta_version != version:
+#    print 'Warning:: Update the version in metainfo.ini !!'
+#print pkg_name,': version =',version
 
 # Scons build directory
-build_prefix= "build-scons"
+#build_prefix= "build-scons"
 
 if sys.platform.startswith('win'):
     install_requires = ["boost", "qhull"]#, "qt4"]
-    #setup_requires = install_requires +['qt4-dev'] 
-    install_requires = [ binary_deps(i) for i in install_requires ]
+    #setup_requires = install_requires +['qt4-dev']
+    #install_requires = [ binary_deps(i) for i in install_requires ]
 #else:
 install_requires = []
 setup_requires = []
-#if sys.platform.startswith('win'): 
+#if sys.platform.startswith('win'):
     #setup_requires.append(["bisonflex"])
 
 pylint_dir = os.path.join('src', 'plantgl')
 
+name = 'VPlants.PlantGL'
+pkg_name = 'openalea.plantgl'
+description= 'PlantGL package for OpenAlea.'
+long_description= 'Plant Geometric Library is a powerfull library to create and display vegetal 3D scene.'
+authors = 'Frederic Boudon, Christophe Pradal'
+authors_email= 'frederic.boudon@cirad.fr, christophe.pradal@cirad.fr'
+url= 'http://openalea.gforge.inria.fr/dokuwiki/doku.php?id=packages:visualization:plantgl:plantgl'
+license = 'Cecill-C'
+
 # Main setup
 setup(
     name=name,
@@ -50,34 +60,34 @@ setup(
     # Define what to execute with scons
     # scons is responsible to put compiled library in the write place
     # ( lib/, package/, etc...)
-    scons_scripts = ['SConstruct'],
-    scons_parameters = ["build_prefix="+build_prefix],
+    # scons_scripts = ['SConstruct'],
+    # scons_parameters = ["build_prefix="+build_prefix],
 
     namespace_packages = ["openalea", "vplants"],
     create_namespaces = True,
 
     py_modules = ["pgl_postinstall",],
     # pure python  packages
-    packages= [ "vplants", 
+    packages= [ "vplants",
                 "vplants.plantgl",
-                pkg_name, 
-                pkg_name+'.math', 
-                pkg_name+'.scenegraph', 
+                pkg_name,
+                pkg_name+'.math',
+                pkg_name+'.scenegraph',
                 pkg_name+'.algo',
-                pkg_name+'.oaplugins', 
-                pkg_name+'.gui', 
+                pkg_name+'.oaplugins',
+                pkg_name+'.gui',
                 pkg_name+'.gui3',
                 pkg_name+'.wralea',
-                pkg_name+'.wralea.demos', 
-                pkg_name+'.wralea.edition', 
-                pkg_name+'.wralea.objects', 
-                pkg_name+'.wralea.operator', 
-                pkg_name+'.wralea.visualization', 
+                pkg_name+'.wralea.demos',
+                pkg_name+'.wralea.edition',
+                pkg_name+'.wralea.objects',
+                pkg_name+'.wralea.operator',
+                pkg_name+'.wralea.visualization',
                 pkg_name+'.wralea.dresser',
                 pkg_name+'.ext',
                 pkg_name+'.codec'
                 ],
-    
+
     # python packages directory
     package_dir= { pkg_name : pj('src','plantgl'),
                    '' : 'src',
@@ -89,9 +99,9 @@ setup(
     zip_safe = False,
 
     # Specific options of openalea.deploy
-    lib_dirs = {'lib' : pj(build_prefix, 'lib'),},
-    bin_dirs = {'bin':  pj(build_prefix, 'bin'),},
-    inc_dirs = { 'include' : pj(build_prefix, 'include') },
+    #lib_dirs = {'lib' : pj(build_prefix, 'lib'),},
+    #bin_dirs = {'bin':  pj(build_prefix, 'bin'),},
+    #inc_dirs = { 'include' : pj(build_prefix, 'include') },
     share_dirs = { 'share' : 'share'},
     postinstall_scripts = ['pgl_postinstall',],
 
@@ -99,10 +109,10 @@ setup(
     # Scripts
     #entry_points = { 'gui_scripts': [ 'pglviewer = openalea.plantgl:start_viewer',]},
     entry_points = {
-        "wralea": ['plantgl.edition = openalea.plantgl.wralea.edition', 
-                   'plantgl.objects = openalea.plantgl.wralea.objects', 
-                   'plantgl.visualization = openalea.plantgl.wralea.visualization', 
-                   'plantgl.operator = openalea.plantgl.wralea.operator', 
+        "wralea": ['plantgl.edition = openalea.plantgl.wralea.edition',
+                   'plantgl.objects = openalea.plantgl.wralea.objects',
+                   'plantgl.visualization = openalea.plantgl.wralea.visualization',
+                   'plantgl.operator = openalea.plantgl.wralea.operator',
                    'plantgl.dresser = openalea.plantgl.wralea.dresser',
                    'plantgl.demos = openalea.plantgl.wralea.demos',
                    ],
@@ -117,9 +127,10 @@ setup(
         'oalab.interface': [
             'PlantGLOAInterfacePlugin = openalea.plantgl.oaplugins.interfaces:PlantGLOAInterfacePlugin',
             ],
+        'console_scripts': [],
               },
 
- 
+
     # Dependencies
     setup_requires = setup_requires + ['openalea.deploy'],
     install_requires = install_requires,
@@ -130,4 +141,4 @@ setup(
 
 
 
-    
+
